internal/common/vote/vote.go

```diff
@@ -0,0 +1,191 @@
+package vote
```

Comment:
We use `shared.ClientID` to refer to islands/clients--please change to reflect that? :) Don't use 1-6